### PR TITLE
Initial README revision

### PR DESCRIPTION
**Pull request by yogeshbdeshpande**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+The API interface is based on the Challenge/Response Interaction Models as documented in [challenge-response](https://github.com/veraison/docs/tree/main/api/challenge-response)
+
+
+## Veraison Trusted Services
+
+Veraison Trusted Services(VTS) provides core services to the Verification and Provisioning front end services. On Provisioning path, it receives Endorsements and Trust Anchors, from [Provisioning](#Provisioning) front end, invokes attestation format specific scheme logic (example PSA) to generate specific keys and values and saves the information in the Endorsement Store and Trust Anchor Store. On the Verification path, it invokes attestation format specific scheme logic to parse attestation token to extract claims. From the claims, it constructs logical index/key to fetch the required trust anchors to validate the token. Subsequently from claims it constructs keys to fetch the associated Endorsements (`golden values`) from Endorsement Store to perform the appraisal of evidence. Upon evaluation it populates the Attestation Results to be communicated to the [Verification](#Verifcation) front end.
```
Comment (setrofim) [inline README.md:24]:
```suggestion
Veraison Trusted Services (VTS) backend provides core services to the Verification and Provisioning frontends. On the Provisioning path, it receives Endorsements and Trust Anchors, from the [Provisioning](#Provisioning) frontend, invokes attestation format (e.g. PSA) specific logic (a "scheme") to generate Endorsements and Trust Anchors, and saves them in the corresponding stores. On the Verification path, it invokes attestation format specific logic (a "scheme") to parse attestation token and extract evidence claims. From these claims, it constructs a logical index/key to fetch the trust anchors required to validate the token, and the associated Endorsements (`golden values`) used to appraise the evidence claims. Upon evaluation, it populates the Attestation Result and communicates it to the [Verification](#Verifcation) frontend.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+
+
+More details about the VTS can be found under [VTS](https://github.com/veraison/docs/tree/main/architecture/verifier#vts)
+
+## KVStore
+
+The key-values store is Veraison Storage Layer. It is used to store both Endorsements and Trust Anchors, as key and values.
```
Comment (setrofim) [inline README.md:31]:
```suggestion
The key-values store is the Veraison Storage Layer. It is used to store both Endorsements and Trust Anchors.
```

**Review by setrofim — CHANGES_REQUESTED** [review]
On a general note --  I'm confused by the capitalisation of terms. It seems inconsistent. E.g. "Attestation Evidences claims" but "Attestation Verification Results"  (both on line 15). Also "Attestation Result" is capitalised, but "attestation format" is not. Am I missing an underlying pattern?

**Comment by yogeshbdeshpande** [comment]
> On a general note -- I'm confused by the capitalisation of terms. It seems inconsistent. E.g. "Attestation Evidences claims" but "Attestation Verification Results" (both on line 15). Also "Attestation Result" is capitalised, but "attestation format" is not. Am I missing an underlying pattern?

No pattern, key words used are capitalized. for consistency, I have changed Attestation Format. Will check again for consistency.

**Review by setrofim — APPROVED** [review]

**Pull request merged**